### PR TITLE
feat: add CODE_OF_CONDUCT.md

### LINKED ISSUE #84: [DOCS] CODE OF CONDUCT
### Description

Kindly create a CODE_OF_CONDUCT.md file in this repo. CODE_OF_CONDUCT is missing here.
I'll be happy if I'm assigned to do so.

### Screenshots

_No response_

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by Susmita-Dey**
**I have added the CODE_OF_CONDUCT file via this PR. 
This closes #84 .**

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/good-first-issue-finder/pull/85"></a>

### CODE REVIEW

**Comment by KendallDoesCoding** [comment]
Thanks @schmelto ... Was just gonna tell that lol

**Review by schmelto — APPROVED** [review]
lgtm

**Review by KendallDoesCoding — APPROVED** [review]

**Comment by KendallDoesCoding** [comment]
> Thanks @schmelto ... Was just gonna tell that lol

@eddiejaoude To link the issue to the PR, Tom editted the messege and linked it

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+an individual is officially representing the community in public spaces.
+Examples of representing our community include using an official e-mail address,
+posting via an official social media account, or acting as an appointed
+representative at an online or offline event.
+
+## Enforcement
+
+Instances of abusive, harassing, or otherwise unacceptable behavior may be
+reported to the community leaders responsible for enforcement at
+susmitadey475@gmail.com.
```
Comment (KendallDoesCoding) [inline CODE_OF_CONDUCT.md:63]: Your not supposed to put your email here you can either put Eddie's email or some support email or none at all. Thanks.

Comment (eddiejaoude) [reply CODE_OF_CONDUCT.md:63]: Good spot, thank you 👍

Comment (eddiejaoude) [reply CODE_OF_CONDUCT.md:63]: Someone can raise a new issue and new PR, more green squares 🟩

Comment (eddiejaoude) [reply CODE_OF_CONDUCT.md:63]: Or my contact page could be used for CoC http://eddiejaoude.io/contact

Comment (KendallDoesCoding) [reply CODE_OF_CONDUCT.md:63]: @eddiejaoude Should I update it? I put community@eddiehub.org in my PR and it's been merged.

Comment (KendallDoesCoding) [reply CODE_OF_CONDUCT.md:63]: Yup, as part of https://github.com/EddieHubCommunity/LinkFree/issues/1099... I would suggest this gets done in every single repo in the organization, but it won't take that long once you've edited the first time and added the security policy

Comment (Susmita-Dey) [reply CODE_OF_CONDUCT.md:63]: Is there any new pr created or shall I create one?

Comment (Susmita-Dey) [reply CODE_OF_CONDUCT.md:63]: Where to give the website contact?

**Pull request merged**